### ISSUE TITLE
🐛 `organizeImports` in vue SFC throws false warnings

### ISSUE THREAD

**Issue by cobsen** [biomejs/biome#8596]
### Environment information

```block
CLI:
  Version:                      2.3.10
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           linux

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           v22.4.1
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         yarn/1.22.22

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  false
  HTML full support enabled:    true

Workspace:
  Open Documents:               0
```

### What happened?

When using the `organizeImports` source action with groups separated with a `:BLANK_LINE:` biome gets confused when a document only contains imports for the 2nd group. This results in biome throwing `The imports and exports are not sorted.` despite them being so.

I have created [a basic repo to easier reproduce](https://github.com/cobsen/biome-repro-1762177411003) this error.

### Expected result

Biome not complaining when the imports are already sorted.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by hwkr** [comment]
Same issue here with blank lines in an SFC.

**Comment by cobsen** [comment]
2.4.14 (more concrete #10092) seems to have fixed this issue
